src/topology/algebra/constructions.lean

```diff
@@ -81,4 +81,12 @@ continuous_induced_dom
 @[to_additive] lemma continuous_coe : continuous (coe : Mˣ → M) :=
 (@continuous_embed_product M _ _).fst
 
+@[to_additive] protected lemma continuous_iff {f : X → Mˣ} :
+  continuous f ↔ continuous (coe ∘ f : X → M) ∧ continuous (λ x, ↑(f x)⁻¹ : X → M) :=
+by simp only [inducing_embed_product.continuous_iff, embed_product_apply, (∘), continuous_prod_mk,
+  op_homeomorph.symm.inducing.continuous_iff, op_homeomorph_symm_apply, unop_op]
+
+@[to_additive] protected lemma continuous_inv : continuous (inv : Mˣ → M) :=
+(units.continuous_iff.1 continuous_id).2
```

Comment:
This feels dubious to me - `inv` isn't simp-normal form, and we shouldn't be using it anywhere. Perhaps
```suggestion
@[to_additive] protected lemma continuous_coe_inv : continuous (λ x : Mˣ, ↑x⁻¹) :=
(units.continuous_iff.1 continuous_id).2
```